apps/web/src/pages/Profile/AccountSettings/RecruitmentAvailability.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from "react";
+import { useIntl } from "react-intl";
+
+import { PoolCandidateStatus, User } from "@gc-digital-talent/graphql";
+import { unpackMaybes } from "@gc-digital-talent/helpers";
+import { Link, Well } from "@gc-digital-talent/ui";
+
+import ApplicationCard from "~/pages/ProfileAndApplicationsPage/components/TrackApplications/ApplicationCard";
+import useRoutes from "~/hooks/useRoutes";
+
+const inlineLink = (href: string, chunks: React.ReactNode) => (
+  <Link href={href} color="black">
+    {chunks}
+  </Link>
+);
+
+const RecruitmentAvailability = ({ user }: { user: User }) => {
+  const intl = useIntl();
+  const paths = useRoutes();
+
+  const activeApplications = unpackMaybes(user.poolCandidates).filter(
+    ({ status }) =>
+      status &&
+      [
+        PoolCandidateStatus.QualifiedAvailable,
+        PoolCandidateStatus.QualifiedUnavailable,
+        PoolCandidateStatus.QualifiedWithdrew,
+        PoolCandidateStatus.PlacedTentative,
+        PoolCandidateStatus.PlacedCasual,
+        PoolCandidateStatus.PlacedIndeterminate,
+        PoolCandidateStatus.PlacedTerm,
+        PoolCandidateStatus.Expired,
+      ].includes(status),
```

Comment:
Not even sure about this but, this looks almost identical to the `QUALIFIED_STATUSES` array outside of expired. The thing I am wondering about is, how is "expired" considered active? :thinking: To me that seems to be contradictory.

Reply:
I'm not sure right now either, I moved this directly over from the Account and Privacy section of the Personal Information page, so I'm not sure. I'll have to ask cause I can't remember if we want expired application cards to appear in this section.

Reply:
@tristan-orourke Do you know what the right logic is here? I know you mentioned that expired is legacy so we don't really care about it?

Reply:
Maybe its worth adding a comment about that

Reply:
Okay 👍 Fixed by [0982b05](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9971/commits/0982b05a0bca56a803d8fc74f7a5d701904bcea8)